app/src/main/java/org/openobservatory/ooniprobe/activity/RunningActivity.java

```diff
@@ -47,7 +48,7 @@
     LottieAnimationView animation;
     private AbstractSuite testSuite;
     private boolean background;
-    private Integer runtime;
+    private static Integer runtime;
```

Comment:
Why is this now static?

Reply:
Because otherwise I couldn't use  in a function below

Reply:
Static variables are very bad, can we avoid using them?